CONTRIBUTING.md

```diff
@@ -1,13 +1,35 @@
 # Contributing
 
-## Set up Git Hooks
-Run the following command to set up git hooks for the project. 
+## Prerequisites
+The following will need to be installed on your device in order to contribute to this project.
```

Comment:
I'd replace the following as it is not a need for functionality:
The following ~will need~ should be ...